tests/dao_tests/test_questionnaire_response_dao.py

```diff
@@ -847,6 +868,38 @@ def test_february_cope_survey(self):
         self.assertEqual(QuestionnaireStatus.SUBMITTED, participant_summary.questionnaireOnCopeFeb)
         self.assertEqual(num_completed_ppi_after_setup + 1, participant_summary.numCompletedPPIModules)
 
+    def test_cope_first_minute_survey(self):
+        """Make sure the dao fills in the summary data for the first COPE minute survey"""
+        self.insert_codes()
+        participant = self.data_generator.create_database_participant(participantId=1, biobankId=2)
+        self._setup_participant()
+
+        questionnaire = self._create_questionnaire(module_code=self.vaccine_1_survey_module_code)
+        authored_date = datetime.datetime(2021, 3, 4)
+        self.submit_questionnaire_response(
+            participant_id=to_client_participant_id(participant.participantId),
+            questionnaire_id=questionnaire.questionnaireId,
+            authored_datetime=authored_date
+        )
+
+        # Check that the summary fields have the survey data
+        summary: ParticipantSummary = self.session.query(ParticipantSummary).filter(
+            ParticipantSummary.participantId == participant.participantId
+        ).one()
+        self.assertEqual(QuestionnaireStatus.SUBMITTED, summary.questionnaireOnCopeVaccineMinute1)
+        self.assertEqual(authored_date, summary.questionnaireOnCopeVaccineMinute1Authored)
+
+        participant_res_data = self.make_participant_resource(participant.participantId)
+        vaccine_module_data = None
+        for module_data in participant_res_data['modules']:
```

Comment:
For future reference:  There is a convenience method in the mixin where you can do something like:
              vaccine_module_data = self. get_generated_items(participant_res_data['modules'], 
                                                         item_key='module', 
                                                         item_value=COPE_VACCINE_MINUTE_1_MODULE_CODE)
                                                         
                returns a list of any entries for the specified module key, so may need to dereference first element.

Reply:
Thanks! I was looking for something like that, but must have not been looking closely enough.